src/nupic/algorithms/SpatialPooler.hpp

```diff
@@ -329,6 +329,11 @@ namespace nupic {
            */
           virtual UInt persistentSize() const;
 
+
+          UInt getPrecision() const {
```

Comment:
hi, thanks,

it should be used eg here:
https://github.com/numenta/nupic.core/pull/629/files#diff-bb28034c282f21e23d3925f2a775db53R578

(I should fix that is I went only with the hard-coded number there. Could
be a macro too. )
and in the compatibility tests, where c++ & py must have the same precision
set.